pkg/model/sdk_helper.go

```diff
@@ -254,6 +254,9 @@ func getMemberByPath(
 	elements := strings.Split(path, ".")
 	last := len(elements) - 1
 	for x, elem := range elements {
+		if elem == "" {
+			continue
+		}
```

Comment:
Yeah, this is fine for now. I suppose we could also warn or panic if the `..` notation was used to indicate a non-list/map field but I'm cool just ignoring for now.

Reply:
Actually, this check is required since the path string in case of a list is currently split into 3 elements and we need to ignore the second one - 

input Path - `ScalableTargets..CreationTime`
the elements array in line 254 - ["ScalableTargets", "", "CreationTime"]

and we need to make sure we don't process the second elem. Another option would be to directly parse for ".." and handle lists but that seemed unnecessary at the moment.

Reply:
@jaypipes just making sure we are on the same page 👆🏻

Reply:
Yep, we're all good.